docker/run_tests_python3.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script marks all the tests we know work in Python 3.
+#
+# Usage: ./docker/run_tests_python3.sh
+
+# This is the list of known working tests by directory/filename. When you
+# have tests in a directory/file working, add it to this list as a new line.
+WORKING_TESTS=(
+    socorro/unittest/lib/test_ooid.py
+)
+
+pytest ${WORKING_TESTS[@]}
```

Comment:
I think this is pretty slick. It only tests what we tell it to test explicitly--no test discovery happens. That'll let us build things up over time.

It can take files or directories or even specific tests/classes (as long as the module loads).